deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -150,23 +154,23 @@ services:
       - OCI_API_KEY
 
   # OCI Stream
-  stream:
-    image: iad.ocir.io/oracle/ateam/mushop-stream:1.0.0
-    hostname: stream
-    restart: always
-    cap_drop:
-      - all
-    cap_add:
-      - NET_BIND_SERVICE
-    read_only: true
-    environment:
-      - reschedule:on-node-failure
-      - OCI_TENANT_ID
-      - OCI_COMPARTMENT_ID
-      - OCI_REGION
-      - OCI_USER_ID
-      - OCI_FINGERPRINT
-      - OCI_API_KEY
+  # stream:
```

Comment:
Users can enable this. the default config is to be able to run locally, so this container would simply not be able to do that without the appropriate resources (stream) created.

Reply:
Can we just leave uncommented, since these vars are in the env pre-requisites anyway? One note about `shipping` and `stream` is that they should also include `STREAM_NAME`